.gitignore

```diff
@@ -2,3 +2,4 @@ vendor/*
 .idea
```

Comment:
Not part of the changes, but this line should be removed, and you should use a global git ignore file ([configuration example here](https://github.com/greg0ire/dotfiles/blob/master/gitconfig#L33)). This way, you don't have to ignore it on all projects.